### ISSUE TITLE
Compilation issue when building locally and on Bitrise intermttently

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/firefox-ios#11163]
I have seen this issue when importing Glean many times both on Bitrise and when building/running tests locally:

```
/Users/vagrant/Library/Developer/Xcode/DerivedData/Client-gwatsyujoxlvmdbdntpwfkcijybc/SourcePackages/checkouts/rust-components-swift/swift-source/all/Generated/Metrics/Metrics.swift:11:8: no such module 'Glean'
import Glean
       ^
** BUILD FAILED **
```

It is not 100% reproducible but I would say once every day it is possible to see this issue.

See some example in this build:
https://app.bitrise.io/build/909d43df-8cbb-4a96-993a-c6aeb7cf715f

cc nbhasin2 have you seen it, any idea why this could happen randomly?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4484)

**Comment by isabelrios** [comment]
Yes, I have seen it on Bitrise too. We are using Intel for all the workflows now

**Comment by isabelrios** [comment]
I pasted one build with the failure in the description, I will log other builds I see with same failure.

**Comment by isabelrios** [comment]
PR build: https://app.bitrise.io/build/3ba1108b-34ff-464c-816a-b9e56523a9a5

**Comment by luser** [comment]
I can reliably reproduce this on my M1 Macbook Pro:
macOS 12.6
Xcode 14.2

**Comment by luser** [comment]
This error occurs while building the [rust-components-swift](https://github.com/mozilla/rust-components-swift) library. While the `Client` project does seem to directly require `Glean` as a Swift Package [in its Xcode project](https://raw.githubusercontent.com/mozilla-mobile/firefox-ios/main/Client.xcodeproj/project.pbxproj):
```
435C85EE2788F4D00072B526 /* XCRemoteSwiftPackageReference "glean-swift" */,
```

…I don't see any such reference in [the `rust-components-swift` `Package.swift`](https://github.com/mozilla/rust-components-swift/blob/90a4bb34e6205d25fa6f67a04f71d9a6f31a91f4/Package.swift). There is an open pull request on that repository that looks related: https://github.com/mozilla/rust-components-swift/pull/64 .

**Comment by isabelrios** [comment]
That was an old issue that we are not experiencing anymore.
We are using Xcode 14.1 it may be a dependency that needs to be updated on 14.2. We will take a look at that upgrade although I think people in the team are already using xcode 14.2 and it works successfully (cc @clarmso) 
Are you running `./bootstrap --force` before building?

**Comment by luser** [comment]
>  Are you running ./bootstrap --force before building?

This was on a fresh build after cloning the repo. I can try using that command to see if it still reproduces.

**Comment by clarmso** [comment]
We should run `./bootstrap` after cloning or updating the repository.

**Comment by lmarceau** [comment]
@luser You should make sure to go through the [Building the code](https://github.com/mozilla-mobile/firefox-ios#building-the-code) section in the readme. Thanks!

**Comment by zengsq** [comment]
same issue, how to fix it?

**Comment by zengsq** [comment]
> @luser You should make sure to go through the [Building the code](https://github.com/mozilla-mobile/firefox-ios#building-the-code) section in the readme. Thanks!

not work
